### ISSUE TITLE
Option for default language

### ISSUE THREAD

**Issue by PCJones** [streamyfin/streamyfin#36]
**Describe the solution you'd like**
An option in the settings for the default audio language track that should be selected if available.

**Additional context**
Currently the default language is not defined or always seems to be English. A lot of German users have dual language files, so it would be great to have the option to select another language as default. 

It might be overkill, but a truly great upgrade for this feature that no other client has would be to select multiple preferred languages in descending order or to have the option to select "Original" as preferred language. The use case for this is that often users might want to prefer English/German in general but Japanese for anime. A basic setting to simply chose a default language is more than enough though.

**Comment by sjorge** [comment]
Yes this would be great, it would also be nice to be able to set this globally and per-library, e.g. one might want different defaults just for their anime library (if they have a separate one)

**Comment by PCJones** [comment]
Being able to specify this per library might be a solution that could be easier to implement then my "advanced" proposal whilst achieving the same. Good thought!

**Comment by fredrikburmester** [comment]
I'm not sure how to do this Jellyfin wise. Are language codes always the same for audio tracks, etc. More information here would be great. Any ideas of implementation logic would also be great.

**Comment by sjorge** [comment]
I'm not sure how the web client does it (its per user there, I know the value is stored in jellyfin.db somewhere) I'll have a look around in the source later but I'm not sure what to look for.
